README.md

```diff
@@ -153,6 +155,26 @@ _Sample `command.json`._
 See the [Command Schema](repo/meta/schema/command-schema.json) for a detailed description of
 the schema.
 
+### Versioning
+
+The registry specification is versioned separately in the
+file `/repo/meta/version.json`.
+
+```json
+{
+  "version": "0.1.0-alpha"
+}
+```
+_Sample `repo/meta/version.json`._
+
+This version is updated with any change to the required file content
+(typically validated using JSON schema) or expected file organization in the
+`repo` directory.
+
+_NOTE: The current version is `0.1.0-alpha` to facilitate rapid
```

Comment:
Might be worth taking out this part, on the basis that we have to update the version in 2 places at the moment (risk of forgetting one).